### DISCUSSION TITLE
how do i fix the camera viewmodel in vehicles.

### DISCUSSION THREAD

**Discussion by Starjet21** [VickyFrenzy/gmod-enhanced-camera-2#27] [General]
ok so this issue is one that has been really bugging me. when I enable "show body in vehicles" I can see my body in a driving position in source vehicles which is good. the issue comes when I try it with simfphys cars where the ViewModel isn't in the right place. what should be happening is that I should see myself in a sitting position in the car while in first person. what actually happens is that I see my character model still in a standing pose (the standing pose for holding a crowbar) and that model is phased halfway through the seat. now granted the original enhanced camera did this too but it was able to fix that by doing a trick with gmod legs 3. that fix doesn't; work with this addon and this issue makes it unusable for things like servers or scenebuilds. is there anyway pof actually fixing this because I really like this addon and don't want to have to replace it. and in case you are wondering this issue even comes up with all other addons disabled so it's not a conflicting addon.

**Comment by VickyFrenzy** [comment]
Hello, you already created an issue about this in #26 
Posting an issue multiple times will not make it fix faster

_Also please read #34_
